homeassistant/components/roomba/irobot_base.py

```diff
@@ -194,6 +200,14 @@ def extra_state_attributes(self):
             state_attrs[ATTR_ERROR] = self.vacuum.error_message
             state_attrs[ATTR_ERROR_CODE] = self.vacuum.error_code
 
+        battery_cycles = self._battery_stats.get("nLithChrg")
+        if battery_cycles == 0:
+            battery_cycles = self._battery_stats.get("nNimhChrg")
+        state_attrs[ATTR_BATTERY_CYCLES] = battery_cycles
```

Comment:
Can this just be
```suggestion
        state_attrs[ATTR_BATTERY_CYCLES] = self._battery_stats.get("nLithChrg") or self._battery_stats.get("nNimhChrg")
```
?